### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 3 - Esercizio 19

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#508] [Esercizi M-Z]
![es.19](https://github.com/user-attachments/assets/3b6c92c0-9e2b-4ba2-a5c0-d86165472ddd)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/1843a487-413d-4152-a76e-e540379be2b1)
